src/Type/Row.purs

```diff
@@ -43,6 +47,8 @@ foreign import kind RowList
 foreign import data Nil :: RowList
 foreign import data Cons :: Symbol -> Type -> RowList -> RowList
 
+data RLProxy (rowList :: RowList) = RLProxy
```

Comment:
To repeat what I said on slack: I'm tempted to leave them out. I think they're only use was cutting down occurrences of `Proxy :: Proxy t`, but the new proxy syntax is better for that.  I've had a look through uses of typelevel-prelude on github and I couldn't see any usages of any of the value-level equivalents.